### DISCUSSION TITLE
not being able to get my student pack again

### DISCUSSION THREAD

**Discussion by iraizo** [community/community#54195] [GitHub Education]
### Hi everyone,

I have verified multiple times now over the years and now they are not accepting my student id for some reason because apparently my github profile isnt filled out, even tho there is my legal name (same on the id, with a date) im not sure what to do now

**Comment by samus-aran** [comment]
Hi @iraizo  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by iraizo** [reply]
![image](https://user-images.githubusercontent.com/35413558/235699921-3faf691d-e6fb-40df-88d5-7864e97e0704.png)
